external/webserver/http_client.c

```diff
@@ -908,17 +908,23 @@ static int prepare_chunk_body(char *buf, int len, const char *body, bool is_last
 
 
 	//append body
-	buflen += snprintf(buf + buflen, HTTP_CONF_MAX_REQUEST_LENGTH - buflen, "%s", body);
+	if(body_len >= HTTP_CONF_MAX_REQUEST_LENGTH - buflen - 2) {
```

Comment:
this condition is same as condition at line 904

Reply:
done